backend/canvas_app_explorer/lti1p3.py

```diff
@@ -16,20 +17,34 @@
 from pylti1p3.contrib.django import DjangoOIDCLogin, DjangoMessageLaunch, \
     DjangoCacheDataStorage, DjangoDbToolConf
 
+
 logger = logging.getLogger(__name__)
 
+
+class CanvasRole(Enum):
+    ACCOUNT_ADMIN = 'Account Admin'
+    SUB_ACCOUNT_ADMIN = 'Sub-Account Admin'
+    TEACHER = 'TeacherEnrollment'
+    STUDENT = 'StudentEnrollment'
+
+
+STAFF_COURSE_ROLES = [CanvasRole.ACCOUNT_ADMIN, CanvasRole.SUB_ACCOUNT_ADMIN, CanvasRole.TEACHER]
```

Comment:
This could be moved to a different file.

Reply:
Done, https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/247/commits/ff39b8e201d5628f8f544f7f7ed44e13d022e347